<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyNotifyItem</class>
 <widget class="QWidget" name="ApplyNotifyItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ApplyNotifyItem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
background-color: #ffffff;</string>
  </property>
  <widget class="QPushButton" name="m_agreePushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_rejectPushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_statusLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>申请中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="m_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QPushButton" name="m_avatarPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="m_nameLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>花未眠</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_operationLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>申请添加你为好友</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
